docs/user-guide/deployments.md

```diff
@@ -454,6 +454,133 @@ nginx-deployment-3066724191   0         0         1h
 Note: You cannot rollback a paused Deployment until you resume it.
 
 
+## Failed Deployment
+
+There will be times your Deployment will get stuck trying to deploy its newest ReplicaSet without ever completing,
```

Comment:
Reword to: 

"Your Deployment may get stuck trying to deploy its newest ReplicaSet without ever completing, due to:
* Insufficient quota
* readiness probe failures
* image pull error
* insufficient permissions
* limit ranges
* application runtime misconfigurations